### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by MoRaad97**
###  In this PR

1. Add Species To The Animals Table Create Function.
2. Insert required data into the table.
3. Add required queries.
4. Add required screenshots.
<hr>

ScreenShot_20220815114006
ScreenShot_20220815114727
ScreenShot_20220815115056
ScreenShot_20220815120707
ScreenShot_20220815124557

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -7,4 +7,5 @@ CREATE TABLE animals (
 	escape_attempts INT NOT NULL,
 	neutered BOOLEAN NOT NULL,
 	weight_kg DECIMAL(5,2) NOT NULL,
+    species VARCHAR(50);
```
Comment (zhadier) [inline schema.sql]: - [x] To add a new column to an existing table, you should use the ALTER TABLE command instead. Most table exist in databases beforehand with lots of stored data so creating a new table to add a new column is not really ideal. Kindly refer to below and add the species by altering the table nstead
```sql
   ALTER TABLE animals ADD species varchar(255);
```

File: queries.sql [hunk 2]
```diff
@@ -7,4 +7,12 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu' ;
...
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+--part 2
+ Select COUNT(*) AS number_of_animals FROM animals;
+ Select escape_attempts, COUNT(*) FROM animals GROUP BY escape_attempts HAVING escape_attempts = 0;
+ SELECT AVG(weight_kg) AS average_weight FROM animals;
+ Select neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
+ SELECT MIN(weight_kg) AS min_weight, MAX(weight_kg) AS max_weight FROM animals;
+ SELECT AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31';
```
Comment (zhadier) [inline queries.sql]: - [x] Kindly ensure that you also add the commands related to the transaction tasks in your queries. Like the one where you're supposed to update animals table and then rollback the change.

**Review by zhadier — COMMENTED** [review]
# Hi @MoRaad97 👋 

Good job so far! 🌤️ 
There are some issues that you still need to work on to go to the next project but you are almost there! 🍓 



## Highlights
✔️ No linter errors
✔️ Github flow was set up correctly
✔️ Professional pull request title and description


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by addod19 — APPROVED** [review]
### Hi @MoRaad97, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [x] Nice job fixing all the comments made by the previous code reviewer :100: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**